server/src/main/xsd/gateway-config-201409.xsd

```diff
@@ -1516,6 +1516,114 @@
         </complexContent>
     </complexType>
 
+    <complexType name="httpProxyServicePropertiesType">
+        <complexContent>
+            <restriction base="gateway:ServicePropertiesType">
+                <sequence>
```

Comment:
I tried using <all> to specify these elements in any order, but it doesn't allow multiple elements like <location-mapping>. So I am using <sequence>. We can live with it or there are two options:
* Remove the changes in schema and use free form for specifying properties (jms follows that)
* Or need to create intermediate elements like <location-mappings> which contains <location-mapping>. 

@jfallows @robinzimmermann Please review the schema change